component/responsive image/image.css

```diff
@@ -0,0 +1,551 @@
+@import url('https://fonts.googleapis.com/css2?family=Redressed&family=Roboto:wght@300&family=Satisfy&family=Shizuru&display=swap');
+
+body{
+font-family: 'Roboto', sans-serif;
+box-sizing: border-box;
+margin: 0px;
+padding: 0px;
+background-color: #111827;
+}
+
+* {
+
+--primary-color: #ecfeff ;
+--secondary-color: #FA0874;
+--black: #111827;
+}

```

Comment:
You can declare variables for color in a different stylesheet and consume it here, it will make your code more clean and readable.

Reply:
will keep in mind from next time thanks